Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3
```

Please use `python:3.9-slim`:

```suggestion
FROM python:3.9-slim
```

https://hub.docker.com/layers/python/library/python/3.9-slim/images/sha256-5d300921213569f955c3954787298127d241c115f31fc92e87305c3e7a71c7ef?context=explore

`python:3.9-slim` compressed size is 40.78 MB
`python:3` size is 325.36 MB